examples/sol-anchor-contract/rust-toolchain

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.70.0"
```

Comment:
This doesn't really fix CI since it is only used for the test build, also imo it'd be better to only have one at the root of the repo.

Reply:
Yeah I didn't make an earnest attempt to fix this yet, was just a first attempt to squash the errors and to come back to it. Will look at it properly soon before merging